packages/core/src/components/card/card.scss

```diff
@@ -11,15 +12,17 @@
   display: block;
 }
 
-:host(:hover) {
-  .dso-card-heading a {
-    color: card.$heading-anchor-hover-color;
-    text-decoration: underline;
-  }
+:host(:focus-within) {
```

Comment:
De `:focus-within` pseudoselector gaat niet werken omdat er ook andere dan `a` interactieve elementen in de host kunnen staan. In dat geval zal de Card onterecht oplichten.

Nu ik dit uittype, heb ik een deja vu 🤔

Reply:
Ik resolve deze omdat ie outdated is en omdat Hans ook al iets aan styling heeft gedaan